zebra-chain/src/parameters/network/subsidy.rs

```diff
@@ -229,8 +251,21 @@ lazy_static! {
     // TODO: Add a reference to lockbox stream ZIP, this is currently based on the number of blocks between the
     //       start and end heights for Mainnet in https://zips.z.cash/draft-nuttycom-funding-allocation
     pub static ref POST_NU6_FUNDING_STREAMS_TESTNET: FundingStreams = FundingStreams {
+        // TODO: Adjust this height range and recipient list once a proposal is selected
         height_range: Height(2_942_000)..Height(3_362_000),
```

Comment:
I think we need to add this numbers as constants so we can have a more clear description on what they are and how/when they will be changed.

Reply:
Moved to constants in https://github.com/ZcashFoundation/zebra/pull/8694/commits/bf4943270366430d218fb6a4900fc5ab76aba7b3.